static/app/views/organizationStats/getReasonGroupName.ts

```diff
@@ -198,7 +164,7 @@ export function getReasonGroupName(outcome: string | number, reason: string): st
     case Outcome.ABUSE:
       return getRateLimitedReasonGroupName(reason as RateLimitedReason);
     case Outcome.FILTERED:
-      return getFilteredReasonGroupName(reason as FilteredReason);
+      return startCase(reason.replace('-', ' '));
```

Comment:
Would this be enough?
```suggestion
      return startCase(reason);
```